src/controller/CHIPDeviceController.cpp

```diff
@@ -2509,7 +2533,7 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
         // This is done in a separate step since we've already used up all the available read paths in the previous read step
         // NOTE: this array cannot have more than 9 entries, since the spec mandates that server only needs to support 9
         // See R1.1, 2.11.2 Interaction Model Limits
-        app::AttributePathParams readPaths[3];
+        app::AttributePathParams readPaths[4];
```

Comment:
But ... we're not putting 4 paths in there, are we?  Why this change?

Reply:
caused by unclean revert, reverted.